packages/cli/src/config/snippets.ts

```diff
@@ -0,0 +1,9 @@
+const BASE_SNIPPETS_URL = "https://snippet.meticulous.ai/";
+
+export const getSnippetsBaseUrl = (): string => {
+  const baseURL =
+    process.env["METICULOUS_SNIPPETS_BASE_URL"] || BASE_SNIPPETS_URL;
+
+  // Append trailing slack if missing from the provided base URL.
+  return baseURL.endsWith("/") ? baseURL : baseURL + "/";
```

Comment:
You don't need this if you use `new URL` to construct them. See other comment.

Reply:
💯 good shout! https://github.com/alwaysmeticulous/meticulous-sdk/pull/84/commits/449da2ca083d48e50b515ffb2a55357a2bb82c35 does this